processes/security_team_members.md

```diff
@@ -15,12 +15,14 @@ policies by PRing their acceptance to this file.
 
 ## Team that triages security reports against node core
 
-- @bnoordhuis - **Ben Noordhuis**
+- @cjihrig - **Colin Ihrig**
 - @indutny - **Fedor Indutny**
-- @rvagg - **Rod Vagg**
 - @jasnell - **James M Snell**
```

Comment:
@jasnell Is https://github.com/nodejs/TSC/issues/599 relevant here? Just confirming.

Reply:
Yep! To be certain, I'm just stepping away from the triage bit.